grpc/build.gradle

```diff
@@ -28,13 +35,11 @@ dependencies {
 sourceSets {
     main {
         proto {
-            srcDirs 'src/main/proto'
-        }
-        java {
-            srcDirs 'src/gen/main/java', 'src/gen/main/grpc', 'src/gen/main/services/java/src'
+            srcDirs 'proto'
```

Comment:
Have you moved proto files into that directory?
You could look at this repo as an example: https://github.com/th2-net/th2-read-db/tree/dev/grpc